### ISSUE TITLE
Most of the Solidity examples in `test/specs/examples` require the `solc-to-k` command that was extracted in kontrol. We should either remove them or bring back some of the `solc-to-k` functionality.

### ISSUE THREAD

**Issue by anvacaru** [runtimeverification/evm-semantics#2338]
Most of the Solidity examples in `test/specs/examples` require the `solc-to-k` command that was extracted in kontrol. We should either remove them or bring back some of the `solc-to-k` functionality.

_Originally posted by @anvacaru in https://github.com/runtimeverification/evm-semantics/issues/2337#issuecomment-1983382860_

**Comment by ehildenb** [comment]
The Solidity files are nice references for folks looking at those specifications, which have been committed. So I don't really see a need to remove them, but we could add a README explaining that.

I also don't want to pull `solc_to_k` back to this repo, as it's Solidity specific and we're trying to move Solidity specific things to Kontrol.

**Comment by ehildenb** [comment]
I don't think this is a bug as much as a documentation issue. And `VERIFICATION.md` may not be a needed file anymore, using KEVM directly for verification is pretty low-level, we should be pushing people towards Kontrol.
